README.md

```diff
@@ -259,6 +259,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Linter for markdown (with presets)](https://github.com/avto-dev/markdown-lint)
 - [Stylelint problem matcher to create annotations](https://github.com/xt0rted/stylelint-problem-matcher)
 - [Run sqlcheck on the PR to identifies anti-patterns in SQL queries](https://github.com/yokawasa/action-sqlcheck)
+- [Run Golint to lint your Golang code](https://github.com/Jerome1337/golint-action)
```

Comment:
Hi @Jerome1337 , I was looking at the readme for this action and feel that the information given is not sufficient for me to be able to utilize it. For example, for the "golint-path" parameter, is that the path where you want the golint command to run? Also, please keep in mind Title Casing for titles, there is a generator found here: https://github.com/sdras/awesome-actions/blob/master/contributing.md

Reply:
Hi @jessicalostinspace I improved the documentation, tell me if some changes are needed.

Reply:
Documentation looks good. Could you please Title Case your titles? There is a formatter in the link I posted above.

Reply:
@jessicalostinspace This should be OK now